src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.cpp

```diff
@@ -363,7 +363,7 @@ extern "C" int32_t AppleCryptoNative_SslGetProtocolVersion(SSLContextRef sslCont
         return errSecParam;
 
     SSLProtocol protocol = kSSLProtocolUnknown;
-    int osStatus = SSLGetNegotiatedProtocolVersion(sslContext, &protocol);
+    OSStatus osStatus = SSLGetNegotiatedProtocolVersion(sslContext, &protocol);
```

Comment:
I'm surprised this didn't result in an error or warning previously... is OSStatus just a typedef for int?